deploy/kubernetes/helm/che/templates/deployment.yaml

```diff
@@ -72,7 +72,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-
+        - name: POD_NAMESPACE
```

Comment:
I think it is better to set `KUBERNETES_NAMESPACE` 
https://github.com/eclipse/che/blob/master/infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/environment/CheInstallationLocation.java#L47

Reply:
I've already seen that code. I used `POD_NAMESPACE` because this environment variable is already used in the Chart unlike `KUBERNETES_NAMESPACE`. Also, in my opinion, pod namespace is more clear in context of deploying a pod than kubernetes namesapce (which is more like a term to me).

Reply:
operator is setting `KUBERNETES_NAMESPACE`